src/form-field/internal.tsx

```diff
@@ -20,13 +20,16 @@ import { joinStrings } from '../internal/utils/strings';
 interface FormFieldErrorProps {
   id?: string;
   children?: React.ReactNode;
+  errorIconLabel?: string;
 }
 
-export const FormFieldError = ({ id, children }: FormFieldErrorProps) => (
+export const FormFieldError = ({ id, children, errorIconLabel }: FormFieldErrorProps) => (
   <div className={styles.error}>
     <div className={styles['error-icon-shake-wrapper']}>
       <div className={styles['error-icon-scale-wrapper']}>
-        <InternalIcon name="status-warning" size="small" />
+        <span role="img" aria-label={errorIconLabel}>
```

Comment:
Can we save ourselves one more HTML element and level of nesting and just add the `role` and `aria-label` to the wrapping `div`?

Reply:
I'm fine with that, I want to double check with @avinashbot since I was following his suggestion, is that agreeable @avinashbot?

Reply:
Good call. @avinashbot 👀  (Just making sure this aligns with the approach you suggested)